llvmlite/tests/test_ir.py

```diff
@@ -234,6 +235,34 @@ def test_metadata_null(self):
         self.assertInText("!{ i32* null }", str(mod))
         self.assert_valid_ir(mod)
 
+    def test_debug_info(self):
+        def order(d):
```

Comment:
This shouldn't be necessary anymore, now that the operands are sorted in add_debug_info(), right?